jekyll/_cci2/certificates.adoc

```diff
@@ -193,8 +193,26 @@ Once you have a valid certificate and key file in `.pem` format, you must upload
 
 . Click "Save" at the bottom of the settings page and restart when prompted
 
-More information is available https://letsencrypt.readthedocs.io/en/latest/using.html#manual[here].
-
 Ensure the hostname is properly configured from the Management Console (`<circleci-hostname>.com:8800`) **and** that the hostname used matches the DNS records associated with the TLS certificates.
 
 Make sure the Auth Callback URL in GitHub/GHE matches the domain name pointing to the Services machine, including the protocol used, for example `**https**://info-tech.io/`.
+
+=== Adding the certificate to Replicated
+
+ Replicated Management Console(`<circleci-hostname>.com:8800`) runs on different web server, so you need to apply the above certificate for Replicated as well.
+
+. To do so, navigate to `https://hostname:8800/console/settings#tls-key-cert`
```

Comment:
Should this be `<circleci-hostname>.com:8800` as above?

Reply:
@rosieyohannan Yes, and there is one more mistake for the url. Basically, hostname includes `.com` or other domain string. So the `<circleci-hostname>.com:8800` name should replace to `<circleci-hostname>:8800`. I'll send a new commit to replace all of them.